contributor/01-introduction-article.md

```diff
@@ -0,0 +1,97 @@
+# Draft comments
+
+ Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)
+* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
+* The [Lowering the Barriers to Entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.
+
+## Outline based Target
+### The Contributor in InnerSource
+
+  - Introduce the speaker. - only required for the video script I'd say. 
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
+
+Outline for this section inspired by the [Trusted Committer introduction article.
+
+## Video Play-book based Target
+How do I best contribute to an InnerSource project?  This segment shares a summary of what it means to be an InnerSource Contributor.  We'll also give an overview of the upcoming training segments on being a great InnerSource Contributor.
+
+
+
+# Article
+
+## The Contributor in InnerSource.
+
+Without contributions, an InnerSource community and its project(s) would not exist in a meaningful way.
+There would be open repositories, ideally equipped with somewhat specific extra documents (e.g. a CONTRIBUTING.md and a README.md) and person(s) committing to them, e.g. a Trusted Committer.
+[EDITOR NOTE:
+- In other articles we draw a distinction between guest and host team.
+- We argue that the Committer is part of the guest team and needs to fit in with the host team.
+- In a more-than-one-person project what do we call the people aside of the TC/PO that are members of the host team? Just team members?
+        - OSS would likely not make this distinction and everyone would end up as contributor until it's committer / trusted committer at some point I guess?
+        - In the ASF-Rule-set this would be the distinction between Developer and Committer.
+]
+
+The Contributor role describes a person that makes contributions to an InnerSource community.
+A contribution can be a code or non-code artifact such as a bug-report, an addition to documentation or a feature request. 
```

Comment:
I think that this introduction should frame a Contributor as someone that is making changes to the contents of a repo.  While true that opening bug or feature issues in the product backlog is a valuable addition, the whole thrust of this section, the learning path, and the InnerSource movement in general is to teach people have to make changes to the contents of a repo that they do not own.  This introductory article should phrase it that way.

[Becoming an InnerSource Contributor](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/cb22be30c19afac949a8bea988a14e805b4fd9c5/contributor/02-becoming-a-contributor.md) gives a small call-out of other ways to contribute, but I don't think we should highlight it up-front.

Reply:
I agree with your point - when bringing this later on we can leave it out in the introduction.
Could you make a commit suggestion from this?

Reply:
Since it's multi-line made a new PR instead: https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/112